HP-637: remove temporary address

NikoHelle opened the pull request:
Removed  temporary address from
- graphQL
- components
- translations

NikoHelle committed:
(HP-637) Removed temporaryAddress from GraphQL and typings

NikoHelle committed:
(HP-637) Removed temporaryAddress from VerifiedPersonalInformation component and tests

NikoHelle committed:
(HP-637) Removed temporary address related translations

akikoskinen reviewed (commented):
Added comments to multiple commits. For some reason Github doesn't seem to list them in this PR.

[1fca5c236](https://github.com/City-of-Helsinki/open-city-profile-ui/commit/1fca5c2360bd95eb0443a84ab4b3c6d0867cd98e#comments)
[68a2e6def](https://github.com/City-of-Helsinki/open-city-profile-ui/commit/68a2e6def3506b19becf7afc06a6c1cb3cb7fb02#comments)
[26fcd340d8](https://github.com/City-of-Helsinki/open-city-profile-ui/commit/26fcd340d87537c2c480489a7a91051229366406#comments)

NikoHelle commented:
> Added comments to multiple commits. For some reason Github doesn't seem to list them in this PR.

Ah, sorry! Fixed the commit messages. "SQ prev" was a note to self to squash it. "VIP" was a typo - should have been "VPI", but I used the long version: "VerifiedPersonalInformation". Also changed the message in translation update.
